pageLayout.css

```diff
@@ -0,0 +1,199 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+:root {
+    --vodka-UI-background-color: white;
+    --vodka-UI-heading-color: #333333;
+    --vodka-UI-text-color: #525252;
+    font-family: Roboto;
+    --highlight-word: black;
+}
+
+::-webkit-scrollbar {
```

Comment:
Suggestion - if you are changing the scrollbar of a section, add the section's class otherwise it will be applied globally.

Reply:
but i have purposely applied to he whole page that is part of my design

Reply:
Okay then, Everything else look's good :)